pkgs/top-level/python-packages.nix

```diff
@@ -16245,6 +16245,28 @@ in modules // {
     };
   });
 
+  pyelftools = buildPythonPackage rec {
+    pname = "pyelftools";
+    version = "0.23";
+    name = "${pname}-${version}";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/p/${pname}/${name}.tar.gz";
+      md5 = "aa7cefa8bd2f63d7b017440c9084f310";
+    };
+
+    # Tests require py27 and py33 at the same time
+    doCheck = false;
```

Comment:
Do all tests require this combination, or some? If it is the latter, can you use only those and disable the others? 
Could you refer to an issue upstream (docs, issue tracker) where this behavior is explained.